test_transpose.py

```diff
@@ -168,25 +182,43 @@ def test_warn_on_convert_to_str_including_column_header(self):
         table = pd.DataFrame({"A": [1, 2], "B": ["x", "y"], "C": [3, 4]})
         result = render(table)
         assert_frame_equal(
-            result["dataframe"],
+            result[0],
             pd.DataFrame({"A": ["B", "C"], "1": ["x", "3"], "2": ["y", "4"]}),
         )
         self.assertEqual(
-            result["error"],
-            (
-                'Headers in column "A" were auto-converted to text.\n'
-                'Column "C" was auto-converted to Text because all columns must '
-                "have the same type."
-            ),
-        )
-        self.assertEqual(
-            result["quick_fixes"],
+            result[1],
             [
                 {
-                    "text": 'Convert "A", "C" to text',
-                    "action": "prependModule",
-                    "args": ["converttotext", {"colnames": ["A", "C"]}],
-                }
+                    "message": i18n_message(
+                        "warnings.headersConvertedToText.error", {"column_name": "A"}
+                    ),
+                    "quickFixes": [
+                        {
+                            "text": i18n_message(
+                                "warnings.headersConvertedToText.quick_fix.text",
```

Comment:
(while we're editing stuff): `.quickFix`, not `.quick_fix`. (I don't have much of a preference of snake-case vs camel-case, but we shouldn't put both in the very same ID.)